config/helm/aws-node-termination-handler/README.md

```diff
@@ -150,12 +150,12 @@ The configuration in this table applies to AWS Node Termination Handler in IMDS
 | `daemonsetTolerations`           | Tolerations for DaemonSet pod assignment. For backwards compatibility the `tolerations` has priority over this but shouldn't be used.                                                                                                                         | `[]`                   |
 | `linuxTolerations`               | Override `daemonsetTolerations` for the Linux DaemonSet.                                                                                                                                                                                                      | `[]`                   |
 | `windowsTolerations`             | Override `daemonsetTolerations` for the Linux DaemonSet.                                                                                                                                                                                                      | `[]`                   |
-| `enableProbesServer`             | If `true`, start an http server exposing `/healthz` endpoint for probes.                                                                                                                                                                                      | `false`                |
+| `enableProbesServer`             | If `true`, start an http server exposing `/healthz` endpoint for probes. Only used in IMDS mode.                                                                                                                                                              | `false`                |
```

Comment:
This is incorrect. The probes server can be exposed whether NTH runs as a daemonset or as a deployment and is not related to IMDS mode. Our default configurations here only enable it for the daemonset, not the deployment, but that is not a technical limitation.

Indeed, our release artifacts explicitly include an aggregated YAML configuration with probes server enabled in Queue Processor mode as a deployment (see the artifact `all-resources-queue-processor.yaml` from any v1 release). Here's the code that generates that file:

https://github.com/aws/aws-node-termination-handler/blob/c3e411b981db6911c668f979a07f25d431cbe073/scripts/generate-k8s-yaml#L61-L66